tests/translator/output/aws-cn/explicit_http_api_with_name.json

```diff
@@ -87,109 +86,110 @@
             "arn:${AWS::Partition}:execute-api:${AWS::Region}:${AWS::AccountId}:${__ApiId__}/${__Stage__}/*",
             {
               "__ApiId__": {
-                "Ref": "MyApiWithIntrinsicName"
+                "Ref": "MyApi"
               },
               "__Stage__": "*"
             }
           ]
         }
-      }
+      },
+      "Type": "AWS::Lambda::Permission"
     },
     "MyApi": {
-      "Type": "AWS::ApiGatewayV2::Api",
       "Properties": {
         "Body": {
           "info": {
-            "version": "1.0",
-            "title": "MyHttpApi"
+            "title": "MyHttpApi",
+            "version": "1.0"
```

Comment:
Why is this changing? I thought our formatting was deterministic?

Reply:
This is a new file, what is changed?

Reply:
Commented before you pushed the new changes, ignore.